[zwFQSUXN] Сжать баннер

AlexeyFrunza opened the pull request:
- сжал фотографию m.png, добавил сжатую m.jpg
- убрал особый функционал у компонента , чтоб не заменялся формат фографии с jpeg на png

Commit:
[zwFQSUXN] Сжать баннер
- [Химчистка] Сжать баннер для мобилки /public/imgs/banners/cleaning/m.png
-- добавил сжатую фотографию m.jpg
-- удалил старую фотографию m.png

Commit:
[zwFQSUXN] Сжать баннер
- [Химчистка] Убрать особый функционал для этой страницы в компоненте UiBanner (path: "/")
-- убрал метод replace в :srcset, который заменял формат изображения
-- тегу source аттрибут type задал "image/jpeg"

weekkkk commented on src/shared/ui/banner/index.vue:
```diff
@@ -100,8 +100,8 @@ onMounted(() => {
       <source :srcset="imgSrc.t" media="(min-width: 768px)" type="image/jpeg" />
       <source
         v-if="route.path === '/'"
-        :srcset="imgSrc.m.replace('jpg', 'png')"
-        type="image/png"
+        :srcset="imgSrc.m"
+        type="image/jpeg"
```
просто убери, полностью, там и так по дефолту `imgSrc.m` используется

Commit:
[zwFQSUXN] Сжать баннер
- [Химчистка] Убрать особый функционал для этой страницы в компоненте UiBanner (path: "/")
-- убрал тег source с особым функционалом
